templates/_templating_scripting.py

```diff
@@ -149,11 +149,51 @@ def prompt_share(args: argparse.Namespace) -> None:
 
     This function is intended for running on a PR on a 'target repo'.
     """
+    pr_number = args.pr_number
+
     def gh_json(sub_command: str, field: str) -> dict:
         command = shlex.split(f"gh {sub_command} --json {field}")
         return json.loads(check_output(command))
 
-    pr_number = args.pr_number
+    # Get all commits in the PR.
+    commits = gh_json(f"pr view {pr_number}", "commits")["commits"]
+
+    # Get all authors found in the commits
+    all_authors = set(
+        commit_author
+        for commit in commits
+        for commit_author in commits
```

Comment:
This gives me the `unhashable type: 'dict'` error. 

I'm not sure if it's the same issue or a different one, but doing `set(`a` for `b` in `c` for `a` in `c`)` seems strange, and doesn't do the same as the original.

Reply:
I had something a bit clunky here, so I "simplified it".
Like I said, coding without ability to test always a bit tricky?

Reply:
Hopefully this fixes !
https://github.com/SciTools/.github/pull/79/commits/34adc4fdbe7f586f15e330e211dc1990341cb201